### PR TITLE
adds reactive vortex armor dont look in the pr's code trust me

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

this adds a reactive vortex hole you get with vortex anomaly cores it has a cool effect you dont need to know what it is

## Why It's Good For The Game

vortex needs an armor

## Changelog
:cl:
add: reactive vortex armor its really cool
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Godspeed king

**Comment by Iamgoofball** [comment]
:+1: :shipit:

**Comment by Shadowflame909** [comment]
maintainers dont look in his code just close the pr

seriously I'm warning you I'm scarred

**Comment by dootdoom** [comment]
based? based on what?

**Comment by tralezab** [comment]
image

**Comment by HomocidlePancakes** [comment]
Speedmerge, please.

**Comment by HomocidlePancakes** [comment]
How many messages can we get before jannies notice?

File: code/modules/clothing/suits/reactive_armour.dm [hunk 1]
```diff
@@ -246,3 +247,27 @@
...
+		var/turf/T = get_turf(owner)
+		for(var/mob/living/A in range(T, 7))
+			visible_message("<span class='notice'>[owner] starts cock-voring [A]!</span>")
+			A.say("NOOOO! HELP ME!!")
+			A.Paralyze(50)
+			if(do_after(owner, 50, TRUE, A))
+				A.say("AAAAAAAAAAAAAAAAAAAA!!")
+				visible_message("<span class='notice'>[A] is sucked into [owner]'s cock, instantly killing them!</span>")
+				A.gib(null, null, TRUE, TRUE)
+		owner.visible_message("<span class='danger'>[src] blocks [attack_text], cock voring the attackers!</span>")
```
Comment (ATH1909) [inline code/modules/clothing/suits/reactive_armour.dm:271]:
```suggestion
		owner.visible_message("<span class='danger'>[src] blocks [attack_text], cock-voring the attackers!</span>")
```

Comment (HomocidlePancakes) [reply code/modules/clothing/suits/reactive_armour.dm:271]: NO THE MAINTAINERS WILL SEE DELETE

Comment (ATH1909) [reply code/modules/clothing/suits/reactive_armour.dm:271]: also, if they're voring them, why not move A inside of owner (instead of gibbing A)?

Comment (LemonInTheDark) [reply code/modules/clothing/suits/reactive_armour.dm]: based

Comment (tralezab) [reply code/modules/clothing/suits/reactive_armour.dm]: image

**Comment by spookydonut** [comment]
cock-vore

**Comment by Fikou** [comment]
> cock-vore

bro i told u to not look at the code wtf?

**Pull request closed without merging**